Source/SelfService/Web/Dockerfile

```
@@ -1,45 +1,26 @@
-FROM node:14.2.0 AS shared-build
+FROM node:14.17.5 AS shared-build
```

What if we go crazy and use a more modren node :)

```suggestion
FROM node:16.7.0 AS shared-build
```

> node --version
v16.7.0